resources/views/auth/_login_default.blade.php

```diff
@@ -0,0 +1,62 @@
+@extends(backpack_view('layouts.auth'))
```

Comment:
Let's move the light/dark switch to the top-right corner of the screen, instead of the top-right corner of the card. As it is now, it kind of attracts attention and makes the card "unbalanced":

![CleanShot 2023-04-19 at 12 03 19@2x](https://user-images.githubusercontent.com/1032474/233025489-bce4fd09-9dd4-41be-aa49-8209ae71c242.png)

Reply:
This applies to the other screens too (register, forgotten password).